src/routes/safe/components/Apps/utils.ts

```diff
@@ -140,3 +134,24 @@ export const uniqueApp =
     })
     return exists ? 'This app is already registered.' : undefined
   }
+
+const canLoadAppImage = (path: string, timeout = 10000) =>
+  new Promise(function (resolve) {
+    try {
+      const image = new Image()
+      image.src = path
+      image.addEventListener('load', function () {
+        resolve(true)
+      })
+      image.addEventListener('error', function () {
+        resolve(false)
+      })
+
+      setTimeout(function () {
```

Comment:
Surprised me not using arrow syntax in this three functions;)

Reply:
copiar y pegar

Reply:
will fix